### ISSUE TITLE
Generate Key Error

### ISSUE THREAD

**Issue by penamerah** [TI-Regceh-2018/Year-Book#4]
Error: php artisan serve   
PHP Warning:  require(C:\Users\ANanda\Documents\GitHub\Year-Book/vendor/autoload.php): Failed to open stream: No such file or directory in C:\Users\ANanda\Documents\GitHub\Year-Book\artisan on line 18

Warning: require(C:\Users\ANanda\Documents\GitHub\Year-Book/vendor/autoload.php): Failed to open stream: No such file or directory in C:\Users\ANanda\Documents\GitHub\Year-Book\artisan on line 18
PHP Fatal error:  Uncaught Error: Failed opening required 'C:\Users\ANanda\Documents\GitHub\Year-Book/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\Users\ANanda\Documents\GitHub\Year-Book\artisan:18
Stack trace:
#0 {main}
  thrown in C:\Users\ANanda\Documents\GitHub\Year-Book\artisan on line 18

Fatal error: Uncaught Error: Failed opening required 'C:\Users\ANanda\Documents\GitHub\Year-Book/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\Users\ANanda\Documents\GitHub\Year-Book\artisan:18
Stack trace:
#0 {main}
  thrown in C:\Users\ANanda\Documents\GitHub\Year-Book\artisan on line 18

What did i do wrong?
Database masih belum di setting karena belum nemu file databasenya. TY

**Comment by Assyatier21** [comment]
Sudah run `npm install` dan `composer install`?

**Comment by penamerah** [comment]
Fixed, ternyata package composer dan npm yg diinstall beda sama yg untuk project.
Im dummy dumb, folder vendor dari 2 package itu ternyata. TY

Tldr; Im dummy dumb for jumping step
